### PR TITLE
Ensure values param are optional

### PR DESCRIPTION
**Pull request by mekwall**
This PR ensures that `getText` values param is optional. It also adds a `.editorconfig` and `lint-staged` that run prettier on committed files.

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -263,7 +263,7 @@ export const getExperimentalCourses = () => ExperimentalCourses
 
 export function getText(
   langKey: string,
-  values: Array<string | number>,
+  values: Array<string | number> | undefined,
```
Comment (gimi-anders) [inline index.ts:266]: Shouldn't it be `values?: Array<string | number`

File: index.ts [hunk 2]
```diff
@@ -263,7 +263,7 @@ export const getExperimentalCourses = () => ExperimentalCourses
...
   textTransform = 'capitalize',
   textStrings: Record<string, string>
```
Comment (gimi-anders) [inline index.ts:268]: Should these two be optional as well?

File: index.ts (hunk 1 above)
Comment (mekwall) [reply index.ts:266]: Since it is followed by more arguments, it cannot be made optional with `?` 😢

Comment (gimi-anders) [reply index.ts:266]: It actually works if you change all parameters like this:
```
  langKey: string,
  values?: Array<string | number>,
  textTransform = 'capitalize',
  textStrings?: Record<string, string>
```

Comment (mekwall) [reply index.ts]: Fixed in b9ae3d4

**Review by gimi-anders — APPROVED** [review]

**Pull request merged**